prow/cluster/jobs/istio/istio/istio.istio.master.yaml

```diff
@@ -713,7 +713,7 @@ postsubmits:
     branches:
     - ^master$
     decorate: true
-    name: integ-galley-local-postsubmit-tests-master
+    name: integ-galley-local-postsubmit-tests-postsubmit-master
```

Comment:
can you modify these to not have postsubmit in the original name? I think there is the same for presubmit

Reply:
Done.